semgrep-core/parsing/Parse_kotlin_tree_sitter.ml

```diff
@@ -355,7 +355,7 @@ let line_string_content (env : env) (x : CST.line_string_content) =
 let return_at (env : env) ((v1, v2) : CST.return_at) =
   let v1 = token env v1 (* "return@" *) in
   let v2 = simple_identifier env v2 in
-  todo env (v1, v2)
+  (v1, Some(v2))
```

Comment:
Note that you don't need parentheses around `v2`. It's more idiomatic to write:
```ocaml
(v1, Some v2)
```

Reply:
ah, got it -- good catch! I'll fix this in my next push 👍